### PR TITLE
Changed the log level of aria2

### PR DESCRIPTION
**Pull request by ArjixGamer**
made aria2 more quiet so it only prints out the download progress and any error, so no useless bloat on your terminal

### CODE REVIEW

**Comment by Blatzar** [comment]
@IguanasInPyjamas any findings from your testing? Should I merge this?

**Comment by IguanasInPyjamas** [comment]
Uhh let me have a look. My download speed hasn't been too bad so I haven't needed it :P

File: anime_downloader/util.py [hunk 1]
```diff
@@ -270,7 +270,7 @@ def format_command(cmd, episode, file_format, speed_limit, path):
     cmd_dict = {
         '{aria2}': 'aria2c {stream_url} -x 12 -s 12 -j 12 -k 10M -o '
                    '{file_format}.mp4 --continue=true --dir={download_dir}'
-                   ' --stream-piece-selector=inorder --min-split-size=5M --referer={referer} --check-certificate=false --user-agent={useragent} --max-overall-download-limit={speed_limit}',
+                   ' --stream-piece-selector=inorder --min-split-size=5M --referer={referer} --check-certificate=false --user-agent={useragent} --max-overall-download-limit={speed_limit}' + ' --console-log-level={}'.format(Config['dl']['aria2_log_level']) if bool(Config['dl']['aria2_log_level']) else '',
```
Comment (Blatzar) [inline anime_downloader/util.py]: `if bool(Config['dl']['aria2_log_level'])`
pretty sure bool() not needed, also I'm pretty sure you can apply it without another string

File: anime_downloader/config.py [hunk 2]
```diff
@@ -20,6 +20,7 @@
         'provider': 'twist.moe',
         'external_downloader': '',
         'aria2c_for_torrents': False,
+        'aria2_log_level': 'notice',
```
Comment (Blatzar) [inline anime_downloader/config.py:23]: Use aria2c or change aria2c_for_torrents for consistency

**Pull request merged**